xapi-profiles-about.md

```diff
@@ -1 +1,138 @@
+this documents aims to address. The challenges are assumed as follows:
+
+1. **Data meant to reflect the same activity often doesn’t match.** As xAPI champions data alignment
+from multiple sources, in practice many products create activity statements that are intended to 
+reflect the same human activity, but are formatted different enough as to prove difficult to 
+analyze correctly. This is because these varying learning activity providers don’t communicate 
+effectively about how they express human activities with xAPI.
+
+2. **Communities of Practice lack shared practices in their definitions of human activity, 
+performance and outcomes.** Communities of Practice at the scale of industry verticals (e.g. 
+military, medical, compliance, trade groups, etc.) have historically approached measurement, 
+even observation, of human activity in subjective, human-based terms. Our xAPI CoPs are often 
+defining and storing their profiles in different ways and at different levels of granularity. 
+Without consistency in how they are written and stored, it is challenging to make use of the 
+profile data. 
+
+Therefore, the goals of the xAPI Profiles Specification are to
+
+* Offer a common way to express controlled vocabularies.
+* Provide instruction on xAPI Statement formation.
+* Describe patterns of xAPI Statements which are meaningful in some way to a profile.
+
+xAPI Profiles rely on linked data technologies to achieve these goals. If you are 
+unfamiliar with linked data, the following article is recommended for all audiences (technical 
+and not-as-technical):
+
+[Linked Data Explained: You’re No Dummy (2015)](https://www.lib.umich.edu/blogs/library-tech-talk/linked-data-explained-you’re-no-dummy)
+
+>“Linked data has been a hot topic lately for both web developers and librarians. 
+>But unless you are highly technical, many of the online definitions for linked data might 
+>make your eyes glaze over and wonder just what exactly is this thing everyone is talking about. 
+>Don’t worry - you are not a dummy. But computers definitely are, which is exactly why we need 
+>linked data on the web...”
+
+## 2.0 How to Use This Document
+
+This is a definitive document which describes how the xAPI Profiles are to be implemented.
+It is a technical document authored specifically for individuals and organizations implementing this 
+technology with the intent of such individuals developing interoperable tools, systems and services that 
+are independent of each other and interoperable with each other. 
+
+Whenever possible, the language and formatting used in this document is intended to be 
+_considerate_ of non-technical readers because various tools, systems and services 
+are based on the specification set described below. For this reason, sections that provide a 
+_high-level overview_ of a given facet of the Experience API are labeled **description** or 
```

Comment:
This still references xAPI itself; also, we don't use these subsections so much in this spec